source/features/dim-bots.tsx

```diff
@@ -15,16 +15,18 @@ const commitSelectors = [
 	'scala-steward',
 	'snyk-bot',
 	'web-flow',
-].map(bot => `.commit-author[href$="?author=${bot}"]`);
-
-commitSelectors.push('.commit-author[href$="%5Bbot%5D"]'); // Generic [bot] label
+];
 
+const commitSelectors = botNames.map(bot => `.commit-author[href$="?author=${bot}"]`);
+commitSelectors.push('.commit-author[href$="%5Bbot%5D"]'); // Generic `[bot]` label in author name
 const commitSelector = commitSelectors.join(',');
 
-const prSelector = [
-	'.opened-by [href*="author%3Aapp%2F"]',
-	'.labels [href$="label%3Abot"]',
-];
+const prSelectors = botNames.map(bot => `.opened-by [title="pull requests opened by ${bot}"]`);
+prSelectors.push(
+	'.opened-by [href*="author%3Aapp%2F"]', // Search query `is:pr+author:app/*`
+	'.labels [href$="label%3Abot"]', // PR tagged with `bot` label
+);
```

Comment:
Kept the more "generic" selectors because they allow hiding Dependabot, Renovate and maybe others without adding them to the list of names.